src/plugins/template/src/plugin.cpp

```diff
@@ -271,38 +271,31 @@ ov::Any ov::template_plugin::Plugin::get_property(const std::string& name, const
     } else if (ov::internal::supported_properties == name) {
         return decltype(ov::internal::supported_properties)::value_type{
             ov::PropertyName{ov::internal::caching_properties.name(), ov::PropertyMutability::RO},
             ov::PropertyName{ov::internal::exclusive_async_requests.name(), ov::PropertyMutability::RW}};
     } else if (ov::available_devices == name) {
         // TODO: fill list of available devices
-        std::vector<std::string> available_devices = {""};
-        return decltype(ov::available_devices)::value_type(available_devices);
+        return decltype(ov::available_devices)::value_type{{""}};
     } else if (ov::device::full_name == name) {
-        std::string device_name = "Template Device Full Name";
-        return decltype(ov::device::full_name)::value_type(device_name);
+        return decltype(ov::device::full_name)::value_type{"Template Device Full Name"};
     } else if (ov::device::architecture == name) {
         // TODO: return device architecture for device specified by DEVICE_ID config
-        std::string arch = get_device_name();
-        return decltype(ov::device::architecture)::value_type(arch);
+        return decltype(ov::device::architecture)::value_type{get_device_name()};
     } else if (ov::device::type == name) {
-        return decltype(ov::device::type)::value_type(ov::device::Type::INTEGRATED);
+        return decltype(ov::device::type)::value_type{ov::device::Type::INTEGRATED};
     } else if (ov::internal::caching_properties == name) {
-        std::vector<ov::PropertyName> caching_properties = {ov::device::architecture};
-        return decltype(ov::internal::caching_properties)::value_type(caching_properties);
+        return decltype(ov::internal::caching_properties)::value_type{ov::device::architecture};
     } else if (ov::device::capabilities == name) {
         // TODO: fill actual list of supported capabilities: e.g. Template device supports only FP32 and EXPORT_IMPORT
-        std::vector<std::string> capabilities = {ov::device::capability::FP32, ov::device::capability::EXPORT_IMPORT};
-        return decltype(ov::device::capabilities)::value_type(capabilities);
+        return decltype(ov::device::capabilities)::value_type{ov::device::capability::FP32,
+                                                              ov::device::capability::EXPORT_IMPORT};
     } else if (ov::execution_devices == name) {
-        std::string dev = get_device_name();
-        return decltype(ov::execution_devices)::value_type{dev};
+        return decltype(ov::execution_devices)::value_type{get_device_name()};
     } else if (ov::range_for_async_infer_requests == name) {
-        // TODO: fill with actual values
-        using uint = unsigned int;
-        return decltype(ov::range_for_async_infer_requests)::value_type(std::make_tuple(uint{1}, uint{1}, uint{1}));
+        return decltype(ov::range_for_async_infer_requests)::value_type{1, 1, 1};
```

Comment:
For some reason `unsigned int` was used here. Can we safely replace it with `int`?

Reply:
It was required for `std::make_tuple` to have correct type could be replaced e.g. by `std::make_tuple<uint>{1,1,1}`.
The `value_type` is tuple of `uint` will work like `std::tuple<unsigned int, unsigned int, unsigned int>{1, 1, 1};`